src/bentoml/_internal/runner/container.py

```diff
@@ -290,13 +291,36 @@ def to_payload(
             return cls.create_payload(
                 plasma_db.put(batch).binary(),
                 batch.shape[batch_dim],
-                {"plasma": True},
+                {"format": "plasma"},
```

Comment:
Let's also remove all of the plasmadb here as well, since the API is going to be removed soon.

Reply:
done